### PR TITLE
fix  white box testing issues for Omnidata spark/hive

### PR DESCRIPTION
**Pull request by Jutao-liu**

### CODE REVIEW

**Review by xulilililili — APPROVED** [review]
lgtm

**Review by tedliUCAS — APPROVED** [review]

File: omnidata/omnidata-hive-connector/connector/src/main/java/org/apache/hadoop/hive/ql/omnidata/reader/OmniDataAdapter.java [hunk 1]
```diff
@@ -193,6 +194,8 @@ public Queue<ColumnVector[]> getBatchFromOmniData() throws UnknownHostException
             } catch (Exception e) {
                 LOGGER.error("OmniDataAdapter getBatchFromOmnidata() has error:", e);
                 failedTimes++;
+            }finally {
+                dataReader.close();
```
Comment (tushengxia) [inline omnidata/omnidata-hive-connector/connector/src/main/java/org/apache/hadoop/hive/ql/omnidata/reader/OmniDataAdapter.java:198]: 注意编码规范，此处需要加空格

File: omnidata/omnidata-spark-connector/connector/src/main/scala/org/apache/spark/sql/execution/ndp/NdpPushDown.scala [hunk 2]
```diff
@@ -446,9 +446,11 @@ object NdpConf {
     val prop = new Properties()
     val inputStream = this.getClass.getResourceAsStream("/"+sourceName)
     if (inputStream==null){
+      inputStream.close()
```
Comment (tushengxia) [inline omnidata/omnidata-spark-connector/connector/src/main/scala/org/apache/spark/sql/execution/ndp/NdpPushDown.scala:449]: 注意编码规范，加空格

File: omnidata/omnidata-spark-connector/connector/src/main/scala/org/apache/spark/sql/execution/ndp/NdpPushDown.scala [hunk 3]
```diff
@@ -446,9 +446,11 @@ object NdpConf {
...
       mutable.Set("")
     }else{
       prop.load(inputStream)
```
Comment (tushengxia) [inline omnidata/omnidata-spark-connector/connector/src/main/scala/org/apache/spark/sql/execution/ndp/NdpPushDown.scala:452]: 注意编码规范，加空格

File: omnidata/omnidata-hive-connector/connector/src/main/java/org/apache/hadoop/hive/ql/omnidata/reader/OmniDataAdapter.java (hunk 1 above)
Comment (Jutao-liu) [reply omnidata/omnidata-hive-connector/connector/src/main/java/org/apache/hadoop/hive/ql/omnidata/reader/OmniDataAdapter.java:198]: 已修复

File: omnidata/omnidata-spark-connector/connector/src/main/scala/org/apache/spark/sql/execution/ndp/NdpPushDown.scala (hunk 2 above)
Comment (Jutao-liu) [reply omnidata/omnidata-spark-connector/connector/src/main/scala/org/apache/spark/sql/execution/ndp/NdpPushDown.scala:449]: 已修复

File: omnidata/omnidata-spark-connector/connector/src/main/scala/org/apache/spark/sql/execution/ndp/NdpPushDown.scala (hunk 3 above)
Comment (Jutao-liu) [reply omnidata/omnidata-spark-connector/connector/src/main/scala/org/apache/spark/sql/execution/ndp/NdpPushDown.scala:452]: 已修复

**Comment by tushengxia** [comment]
> Thanks for your pull request. Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).
> 
> The following commits have not yet signed CLA.
> 
> **[c12b85b](https://github.com/kunpengcompute/boostkit-bigdata/commit/c12b85bc352c964ee785024f3d09cb72adab2dbc)** | add finally **[8f36f9e](https://github.com/kunpengcompute/boostkit-bigdata/commit/8f36f9efa80be0ffd7df7409556d8e0e16532edc)** | close the input stream.
> 
> memo **Please access [here](https://clasign.osinfra.cn/sign/Z2l0aHViJTJGa3VucGVuZ2NvbXB1dGU=) to sign the CLA.**
> 
> It may take a couple minutes for the CLA signature to be fully registered; after that, please reply here with a new comment: **/check-cla** to verify. Thanks.
> 
> * If you've already signed a CLA, it's possible you're using a different email address for your gitee account. Check your existing CLA data and verify the email.
> * If you signed the CLA as an employee or a member of an organization, please contact your corporation or organization to verify you have been activated to start contributing.
> * If you have done the above and are still having issues with the CLA being reported as unsigned, please feel free to file an issue.

/check-cla

**Pull request merged**